directory-connector/src/com/untangle/app/directory_connector/ActiveDirectoryManagerImpl.java

```diff
@@ -267,6 +298,8 @@ public JSONObject getStatusForSettings( ActiveDirectoryServer testServerSettings
                     statusResult = STATUS_RESULTS.FAIL_QUERY_NONE;
                 }else if(e.toString().contains("DSID-0310082F")){
                     statusResult = STATUS_RESULTS.FAIL_QUERY_REFERRAL;
+                }else if(e.toString().contains("error code 49 - 80090308")){
```

Comment:
From a future reliability point of view, instead of entire `"error code 49 - 80090308"`shouldn't we just look for `80090308`, or this number may come for other types of exceptions ?

Reply:
As per [LDAP result code: code 49 ](https://ldap.com/ldap-result-code-reference-core-ldapv3-result-codes/#rc-invalidCredentials)represents the invalid cred error and it combined with 80090308 for generic error, after the that ERROR Number like LdapErr: DSID-XXX is generated. 
So to keep it generic I have choosen this string.